package.json

```diff
@@ -14,11 +14,21 @@
     "build-storybook": "build-storybook"
   },
   "dependencies": {
+    "@mui/icons-material": "^5.10.6"
```

Comment:
I am not sure we'll use these icons. I'm also thinking they should be a peerDependency if/when we do

Reply:
Ok, I'll remove it now and add it later if we need .

Reply:
I'll move it !